sinks/api/http/endpoint.go

```diff
@@ -13,8 +13,26 @@ import (
 	"github.com/go-kit/kit/endpoint"
 	"github.com/ns1labs/orb/pkg/types"
 	"github.com/ns1labs/orb/sinks"
+	"strings"
 )
 
+var restrictiveKeyPrefixes = []string{"secret", "key", "pass", "token"}
```

Comment:
Hey this is reasonable but still depends on this string matching. I was thinking we could instead use the backend feature config which is a structure that actually has detailed information on the sink config:
https://github.com/ns1labs/orb/blob/89f48afdf0c4e541ea3ea14efdfa772639d9a038/sinks/backend/prometheus/prometheus.go#L24

Here we have a type for password:
https://github.com/ns1labs/orb/blob/89f48afdf0c4e541ea3ea14efdfa772639d9a038/sinks/backend/prometheus/prometheus.go#L63

So, we should be able to instead call this CreateFeatureConfig() method for each backend, and elide when the ConfigFeature.Type is "password"?

Reply:
ok, I agree, on it.